CONTRIBUTING.md

```diff
@@ -83,7 +83,18 @@ We follow the [GitHub forking model](https://help.github.com/articles/fork-a-rep
 
 This repository follows a similar approach to other repositories within the [Elastic](https://github.com/elastic) organization, with a few exceptions that make our life easier. One way this repository is simpler is the lack of major version breaking changes. This means we have less backport commits to worry about and makes us a little more productive.
 
-The basic branching workflow we follow for Detection Rules:
+**7.13 and later**
```

Comment:
We should add an `enter` here or `two blank spaces` at the end of the line to format this better. You can see the results of both below. As is, those display on a single line when loaded in markdown.

![image](https://user-images.githubusercontent.com/56412096/122423564-fdaed880-cf5b-11eb-8d57-967e3d35a939.png)

Reply:
Currently - 
![image](https://user-images.githubusercontent.com/56412096/122424535-cb51ab00-cf5c-11eb-9631-007a40d92817.png)

Reply:
++

Reply:
```suggestion
**7.13 and later**

```